src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -181,7 +185,7 @@ public Measure<Velocity<Distance>> calculateVelocity(double targetY, Measure<Ang
 
   public double convertToRPM(double velocity) {
     // 0.0762 meters is diameter of flywheel
-    double circumference = ShooterConstants.FlywheelDiameter * Math.PI;
+    double circumference = ShooterConstants.FlywheelDiameter * Math.PI * 60;
```

Comment:
What is 60 for?

Reply:
to convert to rotations per minute, not rotations per second